fractals/aetherblade/index.md

```diff
@@ -59,11 +66,11 @@ Use <Skill id="31568"/> from your Smokescale pet to stack <Effect name="Stealth"
 ## Lasers <Item id="50082" disableText/>
 
 <Grid>
-<GridItem sm="8">
+<GridItem sm="9">
```

Comment:
GridItems within a grid must add up to 12, else you might see unexpected behaviour. One line can fit exactly 12 `sm`s in total.